.github/workflows/prettier.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  issue_comment:
+    types: [created]
+name: Commands
+jobs:
+  style:
+    if: startsWith(github.event.comment.body, '/style')
+    name: style
+    runs-on: ubuntu-latest
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+      - name: testing
+        run: echo '${{ steps.file_changes.outputs.files_modified}}'
```

Comment:
I won't think we need this as you are running prettier in all the files.

Reply:
ok, I will look into this step.
Thanks for pointing out